### ISSUE TITLE
allow stopping/restarting vehicles via right-click (trac #2755)

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#2755]
related to #1951

Migrated from http://sumo.dlr.de/ticket/2755
```json
{
    "status": "new", 
    "changetime": "2016-12-20T13:57:21Z", 
    "description": "", 
    "reporter": "namdre", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1482242241728103", 
    "component": "simulation gui (GUISIM)", 
    "summary": "allow stopping/restarting vehicles via right-click", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-12-20T13:57:21Z", 
    "milestone": "1.0.0", 
    "owner": "", 
    "type": "enhancement"
}
```

**Comment by namdre** [comment]
can now be accomplished when tracking vehicle via new control [83457915fc5e7c583b855784f26a1aa2b093b100]
However, stopping would still be relevant due to it's effect on lane change behavior by other vehicles

**Comment by namdre** [comment]
fixed via 9ca34ef9cdff723718cfe
